browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3091,6 +3091,24 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_BRAVE_WALLET_CREATE_ACCOUNT_SOLANA_DESCRIPTION" desc="Add account modal create account Solana description">
       Supports SPL compatible assets on the Solana blockchain
     </message>
+    <message name="IDS_EXCLUDED_APPS" desc="Vpn splittunnel text">
```

Comment:
Description is quite important for proper translation, so we should be more specific what text is for. These strings seems to be self explanatory, but it's still a good practice to provide detailed description.